server/auth/store.go

```diff
@@ -1021,7 +1009,7 @@ func (as *authStore) IsAuthEnabled() bool {
 }
 
 // NewAuthStore creates a new AuthStore.
-func NewAuthStore(lg *zap.Logger, be backend.Backend, ci cindex.ConsistentIndexer, tp TokenProvider, bcryptCost int) *authStore {
+func NewAuthStore(lg *zap.Logger, be backend.Backend, tp TokenProvider, bcryptCost int) *authStore {
```

Comment:
could we consider a non-breaking change here?

Reply:
It's neither client/v3/... nor server/embed API ? 

Are you aware about any cases of usage of it outside of etcd ?

If I understand right, it should be part of "internal" package ? 
Also customers (if they do exist) need to adopt due to package `etcd/server/(v3/)auth` rename.